src/webui/www/private/rename_folder.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>QBT_TR(Renaming)QBT_TR[CONTEXT=TorrentContentTreeView]</title>
+    <link rel="stylesheet" href="css/style.css?v=${CACHEID}" type="text/css" />
+    <script src="scripts/lib/mootools-1.2-core-yc.js"></script>
+    <script src="scripts/lib/mootools-1.2-more.js"></script>
+    <script src="scripts/filesystem.js?v=${CACHEID}"></script>
+    <script src="scripts/misc.js?locale=${LANG}&v=${CACHEID}"></script>
+    <script>
+        'use strict';
+
+        new Keyboard({
+            defaultEventType: 'keydown',
+            events: {
+                'Enter': function(event) {
+                    $('renameButton').click();
+                    event.preventDefault();
+                },
+                'Escape': function(event) {
+                    window.parent.closeWindows();
+                    event.preventDefault();
+                },
+                'Esc': function(event) {
+                    window.parent.closeWindows();
+                    event.preventDefault();
+                }
+            }
+        }).activate();
+
+        window.addEvent('domready', function() {
+            const hash = new URI().getData('hash');
+            const oldPath = new URI().getData('oldPath');
+            const name = new URI().getData('name');
+            if (!hash || !oldPath || !name) return;
+
+            const decodedName = decodeURIComponent(name);
+            $('rename').value = decodedName;
+            $('rename').focus();
+            $('rename').setSelectionRange(0, decodedName.lastIndexOf('.'));
```

Comment:
This isn't needed since it's meant to handle file extensions.

Reply:
I changed this to select the entire `name` in this commit: https://github.com/qbittorrent/qBittorrent/pull/13081/commits/124798c06c40b5e7d67d69ba3f4ad3af211bb4db

I also added calls to `focus()` so that when cursor keys are enabled in the browser the focus always returns back to the `rename` input.